shared/devices/device-page/dumb.js

```diff
@@ -1,91 +1,114 @@
 // @flow
-import Render from './index.render'
+import Render from '.'
```

Comment:
we could decide if we want to drive the dumb components directly or use the mock store and drive contained components. i see arguments for both sides. if we make things really dumb based we could use react storybook etc more effectively.